pmm-app/package.json

```diff
@@ -50,6 +50,7 @@
     "lodash": "^4.17.20",
     "moment": "^2.26.0",
     "numeral": "^2.0.6",
+    "pluralize": "^8.0.0",
```

Comment:
What's the real benefit of this library? It's just used in a couple of places, in a trivial way. It looks an overkill and it's possible to replace it with a custom oneliner

Reply:
https://github.com/percona/grafana-dashboards/pull/936#discussion_r480933270

Reply:
Ok, after discussing this, it looks like it's useful also outside of this PR